examples/example-webform/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "example-webform",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "13.0.7",
```

Comment:
I think we should downgrade to next.js 12 and React 17 for now until we have upgraded Next.js Webform to support Next.js 13